packages/amplify_storage_s3/lib/src/S3UploadFile/S3UploadFileOptions.dart

```diff
@@ -17,11 +17,11 @@ import 'package:flutter/foundation.dart';
 import 'package:amplify_storage_plugin_interface/amplify_storage_plugin_interface.dart';
 
 class S3UploadFileOptions extends UploadFileOptions {
-  String targetIdentityId;
+  String? targetIdentityId;
   S3UploadFileOptions(
       {StorageAccessLevel accessLevel = StorageAccessLevel.guest,
-      String contentType,
-      Map<String, String> metadata,
+      String? contentType,
+      Map<String, String>? metadata,
```

Comment:
Does the super need to be updated to be nullable?

Reply:
Yes the UploadFileOptions super class already has nullable contentType and metadata: 

```
class UploadFileOptions extends StorageOptions {
  StorageAccessLevel accessLevel;
  String? contentType;
  Map<String, String>? metadata;

```

Reply:
We updated the Plugin Interfaces to be NS as part of the NS Core PR.  This was to get the Amplify Flutter Module to compile as it has dependencies on all the Plugin Interfaces.